src/NumericalAlgorithms/RootFinding/NewtonRaphson.hpp

```diff
@@ -26,7 +27,8 @@ namespace RootFinder {
  *
  * \requires Function `f` is invokable with a `double`
  * \note The parameter `digits` specifies the precision of the result in its
- * desired number of base-10 digits
+ * desired number of base-10 digits. If this precision is not met after
```

Comment:
Use the `\throws` dox https://www.stack.nl/~dimitri/doxygen/manual/commands.html#cmdthrows